src/main/java/com/google/cloud/genomics/utils/grpc/GenomicsChannel.java

```diff
@@ -103,6 +125,25 @@ public static ManagedChannel fromDefaultCreds() throws SSLException, IOException
    */
   public static ManagedChannel fromOfflineAuth(OfflineAuth auth)
       throws IOException, GeneralSecurityException {
-    return fromCreds(auth.getCredentials());
+    return fromOfflineAuth(auth, null);
+  }
+
+  /**
+   * Create a new gRPC channel to the Google Genomics API, using OfflineAuth or the application
+   * default credentials.
+   *
+   * This library works with both the older and newer support for OAuth2 clients.
```

Comment:
This sentence makes me want to read it several times.  How about the following:

*This library will work with both the newer and older versions of OAuth2 client-side support.*